### DISCUSSION TITLE
Send an email when custom domain integration is ready

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#420] [Anything Else]
This is when trying to set up custom domains in order to serve our script as a first party connection. It would be a nice user experience improvement if we send an email when this is ready so you can change the Javascript snippet on your site. Right now the instructions are to wait and manually check when the script is loading from your subdomain before changing the snippet.

**Comment by Vigasaurus** [comment]
What if we just had some UI similar to how most DNS/"waiting for nameserver" services do it, where instead of an email we just start up a listener on the front end to poll the custom domain as needed, and showing a UI similar to the "waiting for pageview" until the file is verified to be serving correctly (maybe just listening for a 200, or potentially checking the hash of the file received) - say polling every 15 seconds or similar. 

How necessary do you think the email side of it is?

**Reply by metmarkosaric** [reply]
For most sites, this integration goes live within minutes so even something in the UI could work. My idea was basically to give people a smoother experience: "don't worry about manually checking the integration, we'll let you know when it's ready so you can switch the code".

**Reply by ukutaht** [reply]
Yes I think I prefer the web UI over an email. 90%+ cases the DNS record is new which means it propagates within seconds. No long waiting is necessary so better to keep the user in the UI.
